velox/docs/functions/spark/aggregate.rst

```diff
@@ -114,6 +114,22 @@ General Aggregate Functions
 
     Returns b
 
+.. spark:function:: mode(x) -> [same as x]
+
+    Returns the most frequent value for the values within `x`.
+    NULL values are ignored. If all the values are NULL, or
```

Comment:
What if two values occurs same times?

Reply:
For example:
```
SELECT mode(x)
        FROM (
            VALUES
                (0), (0), (10), (10)
        ) AS t(x);
```
The result is 0 or 10. This is indeterminate, as we store intermediate data in a hash map.

Reply:
Thanks for your explanation, can you add it to the document?